service/history/workflowExecutionContext.go

```diff
@@ -699,7 +699,8 @@ func (c *workflowExecutionContextImpl) updateWorkflowExecutionWithNew(
 	// emit workflow completion stats if any
 	if currentWorkflow.ExecutionInfo.State == persistence.WorkflowStateCompleted {
 		if event, err := c.msBuilder.GetCompletionEvent(); err == nil {
-			emitWorkflowCompletionStats(c.metricsClient, domainName, event)
+			taskList := currentWorkflow.ExecutionInfo.TaskList
```

Comment:
if this is for decision, plz also take sticky tasklist into consideration

Reply:
This is for customers to filter. sticky tasklist name does not make sense to them.